### PR TITLE
Setup.sh addition

### LINKED ISSUE #6: Automating the setup
1.  Installing the Pipenv.
2.  Installing software dependencies.
3.  Starting out the application.
When we run the VM.

### PR DESCRIPTION
**Pull request by TomerNewmanPrograms**
-Adding Setup.sh file
-Changing the vagrantfile

In order to automate the setup including:

1. Installing the Pipenv.
2. Installing software dependencies.
3. Starting out the application.

Fixes #6

### CODE REVIEW

**Review by MatanP12 — APPROVED** [review]
MY name is Matan, and I approve this PR

**Comment by abiton1** [comment]
The `setup.sh` file should end with a new line (you can also see a remark in your commit)

**Comment by TomerNewmanPrograms** [comment]
> The `setup.sh` file should end with a new line (you can also see a remark in your commit)

fixed

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ex
+# The -e option would make our script exit with an error if any command
+# fails while the -x option makes verbosely it output what it does
+
+# Install Pipenv, the -n option makes sudo fail instead of asking for a
```
Comment (bregman-arie) [inline setup.sh:5]: It's good that you documented what -n does but another common approach is to specify the longer version of the argument: --non-interactive

**Review by OmerCS8 — APPROVED** [review]
LGTM

**Review by AlonShakaroff — APPROVED** [review]
Looks good to me 👍

**Pull request merged**